src/main.cpp

```diff
@@ -186,100 +208,192 @@ void addPackOptions(po::options_description& desc)
 {
     po::options_description pack("Pack Options");
     pack.add_options()
-        ("type,t", po::value<string>(), "Type of the input model. Currently supported types:\n \t - caffe")
-        ("json,j", po::value<string>(), "Model metadata in JSON format. Command line parameters will override entries in this file if present.")
-        ("name,n", po::value<string>(), "Model name.")
-        ("version,V", po::value<string>(), "Model version.")
-        ("description,d", po::value<string>(), "Model description.")
-        ("labels,l", po::value<string>(), "Labels file name.")
-        ("date-time", po::value<string>(), "Date/time the model was created (optional). Date/time is normally set "
+        ("type,t", po::value<string>()->value_name("TYPE"),
+            "Type of the input model. Currently supported types:\n \t - caffe")
+        ("json,j", po::value<string>()->value_name("PATH"),
+            "Model metadata in JSON format. Command line parameters will override "
+            "entries in this file if present.")
+        ("name,n", po::value<string>()->value_name("NAME"), "Model name.")
+        ("category,C", po::value<string>()->value_name("CATEGORY"), "Model category.")
+        ("version,V", po::value<string>()->value_name("VERSION"), "Model version.")
+        ("description,d", po::value<string>()->value_name("DESCRIPTION"), "Model description.")
+        ("labels,l", po::value<string>()->value_name("PATH"), "Labels file name.")
+        ("label-names", po::value<vector<string>>()->value_name("LABEL1 [LABEL2 ...]"),
+            "A list of label names.")
+        ("date-time", po::value<string>()->value_name("DATE_TIME"),
+            "Date/time the model was created (optional). Date/time is normally set "
```

Comment:
This is "THE" ISO format, at least when programming languages are concerned. I'll use your wording though.